commands/util.go

```diff
@@ -91,7 +91,7 @@ func initSafe() error {
 		return err
 	}
 
-	if _, err := backendClient.Load(); err == nil {
+	if _, err := backendClient.Load(); err == nil { // nolint: vetshadow
```

Comment:
Is this shadowing directive there because there's already a variable named `err` outside the scope of this func? If so, why doesn't the assignment on line 50 have one?

Reply:
It's suppressing the warning for this line. In this case we can safely ignore it.